pkg/constants/query.go

```diff
@@ -14,4 +14,7 @@ const (
 	QueryOpLesserThan         = "$lt"
 	QueryOpGreaterThanOrEqual = "$gte"
 	QueryOpLesserThanOrEqual  = "$lte"
+
+	// Top-level fields allowed in the query
+	Selector = "selector"
```

Comment:
Prefix with `QueryField*` to avoid confusion, same logic as having `QueryOp*` above.
I expect we'll have more fields in the future, like  `QueryFieldLimit` etc.

Reply:
Done